webgpu.yml

```diff
@@ -3291,34 +3299,43 @@ objects:
           TODO
         returns:
           doc: |
-            TODO
+            Returns a mutable pointer to beginning of the mapped range.
+            It may be written, but reading is undefined behavior unless the buffer was also mapped with @ref WGPUMapMode_Read.
```

Comment:
Reading should be a valid for mapped write buffers, because any mapped range should get flushed to the GPU and not be modified if the implementation didn't end up writing to it. That's what happens in JS.

Reply:
I couldn't remember if this was a requirement for native mapping to avoid reading uninitialized memory. I know reading is valid in JS, but that was also because it's currently impossible to modify a MAP_WRITE buffer _except_ from the CPU via mapping, so there can be a CPU-side shadow copy. That shadow copy prevents JS from actually reading from the real map anyway.